docs/streaming-kinesis-integration.md

```diff
@@ -119,15 +119,14 @@ A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or m
 
 3. **Deploying:** As with any Spark applications, `spark-submit` is used to launch your application. However, the details are slightly different for Scala/Java applications and Python applications.
 
-	For Scala and Java applications, if you are using SBT or Maven for project management, then package `spark-streaming-kinesis-asl_{{site.SCALA_BINARY_VERSION}}` and its dependencies into the application JAR. Make sure `spark-core_{{site.SCALA_BINARY_VERSION}}` and `spark-streaming_{{site.SCALA_BINARY_VERSION}}` are marked as `provided` dependencies as those are already present in a Spark installation. Then use `spark-submit` to launch your application (see [Deploying section](streaming-programming-guide.html#deploying-applications) in the main programming guide).
+	You'll first need to perform a custom build of Spark. **Note that by linking to this library, you will include [ASL](https://aws.amazon.com/asl/)-licensed code in your application.** Download Spark source and follow the [instructions](building-spark.html) to build Spark with profile *-Pkinesis-asl*.
```

Comment:
Is this true? if you build the Kinesis code into your app as in the following text and next paragraph it should work.